src/components/Book.js

```diff
@@ -1,20 +1,21 @@
 import { PropTypes } from 'prop-types';
 import { useDispatch } from 'react-redux';
-import { removeBook } from '../redux/books/books';
+import { deleteBook } from '../redux/books/books';
 
 const Book = (props) => {
-  const { id, title, author } = props;
+  // eslint-disable-next-line camelcase
```

Comment:
- [ ] As requested by the previous reviewer, please do not disable linters and work on solving those linter errors. As I see, item_id is a property and you can name it as you want.

Reply:
@medaminedev66 theitem_id belongs to the API call, I can't change, or the functionality will break.